tests/unit/event_callback.c

```diff
@@ -91,20 +108,7 @@ int main(int argc, char *argv[])
 
   driz_ret= drizzle_quit(con);
   ASSERT_EQ_(DRIZZLE_RETURN_OK, driz_ret, "%s", drizzle_strerror(driz_ret));
-
-  // Check that the drizzle_event_callback was called. It is done indirectly by
-  // asserting that cxt_b has been incremented 4 times.
-  // The increments of cxt_b correspond to the following state changes:
-  //
-  // 1. when drizzle connection is initied by calling drizzle_connect
-  // 2. while connecting to server
-  // 3. when server handshake succeeds
-  // 4. when the drizzle_st connection is closed by calling drizzle_quit
-  //
-  // Please note that stepping through the test with 'make gdb-event_callback'
-  // can cause the test to fail
-  ASSERT_EQ_(4, cxt_b, "Unexpected number of event callbacks: Got '%d', \
-    Expected '4'", cxt_b);
+  ASSERT_TRUE(cxt_b >= 3);
```

Comment:
Unfortunately not as the macro is defined. However could do `ASSERT_EQ_(cxt_b >= 3, true, ...)`